admin/modules/tools/maillogs.php

```diff
@@ -301,6 +301,8 @@
 		$log['subject'] = htmlspecialchars_uni($log['subject']);
 		$log['dateline'] = my_date('relative', $log['dateline']);
 
+		$plugins->run_hooks("admin_tools_maillogs_log");
```

Comment:
We might want to pass `$log` as a parameter to this hook I think?

Reply:
@euantorano this hook is in the global scope, so there is no need.